.changes/880-1111.json

```diff
@@ -0,0 +1,5 @@
+{
+    "description": "added a zig-based image (v0.10.0), allowing multiple targets to be built from the same image, using cargo-zigbuild (v0.14.1).",
```

Comment:
I've added the specific versions to the description so these can be modified until the actual 0.3.0 release, and just link every PR associated.